common/presenters/date-table/locations-to-population-table-component.js

```diff
@@ -3,7 +3,70 @@ const { times, keys } = require('lodash')
 
 const tablePresenters = require('../table')
 
-const { dayConfig, establishmentConfig } = require('./population-helpers')
+const {
+  dayConfig,
+  establishmentConfig,
+  headerRowConfig,
+} = require('./population-helpers')
+
+function locationsToPopulationAndTransfersTables({
```

Comment:
This function is very very similar to `locationsToPopulationTable`, and I think that usage should be refactored to be based off this, and that function removed.